frontend/packages/metal3-plugin/src/components/add-host.tsx

```diff
@@ -193,7 +193,7 @@ const AddHost: React.FC = () => {
                 Create
               </Button>
               <Link
-                to={`/k8s/ns/${getActiveNamespace()}/baremetalhosts`}
+                to={`/k8s/ns/${getActiveNamespace()}/${referenceForModel(BaremetalHostModel)}`}
```

Comment:
We use history back for cancel in forms.

You should use the helpers for creating URLs instead of building them yourselves. See

https://github.com/openshift/console/blob/master/frontend/public/components/utils/resource-link.tsx

Reply:
Done